pagopa/pn-frontend-e2e
# PN 9163 e 2 e destinatario PG sezione stato piattaforma disservizi in corso PG

## Short description

Refactor test visualizza disservizio PG

## List of changes proposed in this pull request

Added step to control table row with Stato "in corso"
modified step for PA to match requirement for both PA and PG

## How to test

Verify the correct display of the outage information. Log in helpdesk, create an outage, verify the correct creation of it and verify the correct display of the table in stato della piattaforma PG

- [ x ] Does this PR require a manual test in AWS CodeBuild?
